src/visip/dev/parameters.py

```diff
@@ -124,10 +125,18 @@ def extract_func_signature(func, skip_self=True):
     :param skip_self: Skip first parameter if its name is 'self'.
     :return:
     """
+    from . import dtype_new
+    from ..code import wrap
+
     signature = inspect.signature(func)
     no_value = inspect.Parameter.empty
     parameters = Parameters()
 
+    def none_to_nonetype(type):
```

Comment:
PAssing None as a typehint should be just missing type_hint which is an error but should not be converted as otherwise we can not correctly represent it

Reply:
- prejmenovat ActionParameter.no_default na ActionParameter.empty
- konzistentne upravit, vyponout konverzi z ActionParameter.empty na None v return_type a inti ActionPArameter
   pouzit empty jak defalut pro type_hint a default pro action_workflow._Slot

Reply:
- prejmenovat z ActionPArameter.no_default na ActionPArameter.empty
- zrusit konverze na None v rteunt_type a ActionParameter.__init__
- zkotrolovat volani _Slot a _SlotCall zda se nekde nepredava None